backend/amd64/proc.ml

```diff
@@ -438,7 +438,7 @@ let destroyed_at_terminator (terminator : Cfg_intf.S.terminator) =
   match terminator with
   | Never -> assert false
   | Always _ | Parity_test _ | Truth_test _ | Float_test _ | Int_test _
-  | Return | Raise _ | Tailcall _ ->
+  | Return | Raise _ | Tailcall _ | Poll_and_jump _ ->
```

Comment:
For `Poll_and_jump`, shouldn't it
be `destroyed_at_alloc_or_poll`?

Reply:
Fixed